release/archives/linux/data-prepper-jdk-x64.sh

```diff
@@ -15,13 +15,14 @@ fi
 
 PIPELINES_FILE_LOCATION=$1
 CONFIG_FILE_LOCATION=$2
-DATA_PREPPER_HOME=$(dirname $(realpath $0))
-EXECUTABLE_JAR=$(ls -1 $DATA_PREPPER_HOME/bin/*.jar 2>/dev/null)
+DATA_PREPPER_BIN=$(dirname $(realpath $0))
```

Comment:
nit: I recommend wrapping variables used as args or may return paths in quotes to prevent globbing. As an example here `DATA_PREPPER_BIN=$(dirname "$(realpath "$0")")`